curriculum/challenges/english/04-data-visualization/json-apis-and-ajax/convert-json-data-to-html.english.md

```diff
@@ -24,6 +24,7 @@ json.forEach(function(val) {
 });
 ```
 
+<strong>Note:</strong> For this challenge you need to add new html elements to the page, so you cannot rely on textContent. Instead you need to use innerHTML which can make a site vulnerable to cross site scripting attacks.
```

Comment:
I think it would be better to capitalize HTML and add backticks (`) around `textContent` and `innerHTML`. This will render the two methods as inline code elements and will make it more consistent with other parts of the curriculum:

```
<strong>Note:</strong> For this challenge you need to add new HTML elements to the page, so you cannot rely on `textContent`. Instead you need to use `innerHTML` which can make a site vulnerable to cross site scripting attacks.
```

Reply:
@Wocanilo Once you make the change suggested above on this PR, you will need to change the translation on PR https://github.com/freeCodeCamp/freeCodeCamp/pull/37037 to match.